components/bounty/tabs-sections/item-row/view.tsx

```diff
@@ -121,13 +127,13 @@ export default function ItemRowView({
           />
         ) : (
           <div className="d-block d-sm-none mt-3">
-            <RenderProposalOrPullRequest />
+            <RenderProposalOrDeliverable />
           </div>
         )}
 
         <div className="col-lg-4 col-md d-none d-sm-block">
           <div className="d-flex flex-row gap-3 justify-content-end align-items-center">
-            <RenderProposalOrPullRequest />
+            <RenderProposalOrDeliverable />
```

Comment:
why are we rendering this component twice? :o

Reply:
because of the mobile/web rendering of the design